pkg/api/v1/api.proto

```diff
@@ -305,6 +305,10 @@ service OverviewService {
   rpc StreamOverview (GetOverviewRequest) returns (stream GetOverviewResponse) {}
 }
 
+service EnvironmentService {
```

Comment:
please add a screenshot to the PR description

Reply:
![image](https://github.com/freiheit-com/kuberpult/assets/29933511/ec34bf2a-6c21-49ee-a6d1-8996b4476926)